.github/workflows/win.yml

```diff
@@ -0,0 +1,46 @@
+name: windows
+
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        CONDA_PY: ["36", "37"]
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        auto-activate-base: true
+        activate-environment: ''
+    - name: Install
+      env:
+        CONDA_PY: ${{ matrix.CONDA_PY }}
+      run: |
+        conda config --system --set always_yes yes --set changeps1 no
+        conda config --system --append channels conda-forge
+        conda config --system --append channels esss
+        conda install -c conda-forge conda-devenv conda=4.8.3
+        conda info -a
+        conda devenv
+    - name: Tests
+      run: |
+        conda init powershell
+        conda activate esss-fix-format-py${{ matrix.CONDA_PY }}
+        Get-Item Env:PYTHONPATH
```

Comment:
Let's remove this and try again.

Reply:
OK, I'm on it...

Reply:
Didn't work :(

Reply:
I will try to set `PYTHONPATH` and `PATH` with `env` field in Tests step.